src/Components/Layout/Layout.tsx

```diff
@@ -0,0 +1,15 @@
+import { FC } from 'react';
+import { Outlet } from 'react-router-dom';
+import { NavBar } from '../NavBar';
+
+export const Layout: FC = () => (
+  <>
+    <NavBar />
+
+    <main className="section">
+      <div className="container">
+        <Outlet />
+      </div>
+    </main>
```

Comment:
Would be better to wrap your components in this container to avoid using react fragment

```suggestion
    <main className="section">
        <Outlet />
    </main>
```